_data/people.yml

```diff
@@ -78,7 +78,12 @@
     url: "bl"
     role: "Undergraduate Researcher"
     image: laufenberg.jpg
-
+  - name: "CGrace Kelly"
+    url: "CGK ; )"
```

Comment:
We're really close now!  This URL needs to reference the directory name that we kept `CGK`